examples/testerina-mocking-functions/testerina_mocking_functions.md

```diff
@@ -3,12 +3,13 @@
 Mock functions allow you to hide the real function implementation and engage your own definition when running tests.
 This allows you to isolate your tests from the other modules and functions.<br/><br/>
```

Comment:
```suggestion
This allows you to isolate your tests from the other modules and functions.<br/><br/>
```
We can remove these line breaks now. Please check and remove all the occurrences.